libproc/lustre.c

```diff
@@ -131,6 +131,42 @@ typedef enum {
  * On EOF, functions return -1 with errno clear.
  */
 
+/*
+ * Fill-in the supplied version components with the Lustre version
+ * found in /sys.
+ *
+ * Returns -1 or less on error, 0 or greater on success.
```

Comment:
What does the magnitude of the return value indicate? Why not just -1 or 0?

Reply:
I'm not sure. I was trying to be consistent with code around it.